--- mpd_sima_0.9.1/data/Makefile.wiget	2012-02-26 20:03:31.113116022 +0100
+++ mpd_sima_0.9.1/data/Makefile	2012-02-26 20:04:32.612348816 +0100
@@ -36,7 +36,7 @@ man: simadb_cli mpd_sima mpd_sima.cfg
 
 html: simadb_cli_html mpd_sima_html mpd_sima.cfg_html
 
-pdf: simadb_cli.1 mpd_sima.1 mpd_sima.cfg.4
+pdf: simadb_cli.1 mpd_sima.1 mpd_sima.cfg.5
 	/usr/bin/man -l -Tps mpd-sima.1 | /usr/bin/ps2pdf - ./mpd_sima.pdf
 	/usr/bin/man -l -Tps mpd-sima.cfg.5 | /usr/bin/ps2pdf - ./mpd_sima.cfg.pdf
 	/usr/bin/man -l -Tps simadb_cli.1 | /usr/bin/ps2pdf - ./simadb_cli.pdf
--- mpd_sima_0.9.1/Makefile.wiget	2012-02-26 20:00:43.138878721 +0100
+++ mpd_sima_0.9.1/Makefile	2012-02-26 20:02:30.003688218 +0100
@@ -10,13 +10,13 @@ MANDIR = $(DATADIR)/man
 
 all: man bin
 clean:
-	rm -f mpd-sima.1.gz simadb_cli.1.gz mpd-sima.cfg.1.gz mpd-sima simadb_cli
+	rm -f mpd-sima.1.gz simadb_cli.1.gz mpd-sima.cfg.5.gz mpd-sima simadb_cli
 
-mpd-sima.1.gz:
+mpd-sima.1.gz: data/mpd-sima.1
 	cat data/mpd-sima.1 | gzip > mpd-sima.1.gz
 
-mpd-sima.cfg.1.gz:
-	cat data/mpd-sima.cfg.1 | gzip > mpd-sima.cfg.1.gz
+mpd-sima.cfg.5.gz: data/mpd-sima.cfg.5
+	cat data/mpd-sima.cfg.5 | gzip > mpd-sima.cfg.5.gz
 
 simadb_cli.1.gz:
 	cat data/simadb_cli.1 | gzip > simadb_cli.1.gz
@@ -32,17 +32,18 @@ bash_completion:
 	install -m644 data/mpd_sima.completion.sh $(ETC)/bash_completion.d/
 	mv $(ETC)/bash_completion.d/mpd_sima.completion.sh $(ETC)/bash_completion.d/mpd-sima
 
-man: mpd-sima.1.gz simadb_cli.1.gz mpd-sima.cfg.1.gz
+man: mpd-sima.1.gz simadb_cli.1.gz mpd-sima.cfg.5.gz
 
 bin: mpd-sima simadb_cli
 
 install: man bin
-	install -d $(BIN) $(DATADIR)/mpd-sima $(MANDIR)/man1
+	install -d $(BIN) $(DATADIR)/mpd-sima $(MANDIR)/man1 $(MANDIR)/man5
 
 	install -m755 mpd-sima $(BIN)
 	install -m755 simadb_cli $(BIN)
 
-	install -m644 mpd-sima.1.gz mpd-sima.cfg.1.gz simadb_cli.1.gz $(MANDIR)/man1
+	install -m644 mpd-sima.1.gz simadb_cli.1.gz $(MANDIR)/man1
+	install -m644 mpd-sima.cfg.5.gz $(MANDIR)/man5
 
 	# listing all sub folder to install (prevent a remaining .svn directory to be install)
 	for sourcedir in `find src/ -name .svn -prune -o -type d -print | sed 's:src/::g'` ; do \
@@ -59,8 +60,8 @@ uninstall:
 	rm -f $(BIN)/mpd-sima
 	rm -f $(BIN)/simadb_cli
 	rm -f $(MANDIR)/man1/mpd-sima.1.gz
-	rm -f $(MANDIR)/man1/mpd-sima.cfg.1.gz
 	rm -f $(MANDIR)/man1/simadb_cli.1.gz
+	rm -f $(MANDIR)/man5/mpd-sima.cfg.5.gz
 	rm -f $(ETC)/bash_completion.d/mpd-sima
 	rm -rf $(DATADIR)/mpd-sima
 
