--- mpd_sima_0.8.0/src/utils/startopt.py.wiget	2012-02-03 14:09:03.722512780 +0100
+++ mpd_sima_0.8.0/src/utils/startopt.py	2012-02-03 14:10:09.119982354 +0100
@@ -52,6 +52,13 @@ def is_file_exist(option, opt_str, value
         raise OptionValueError('no access to "%s"' % value)
     setattr(parser.values, option.dest, value)
 
+def is_directory_rwable(option, opt_str, value, parser):
+    value = normalize_path(value)
+    # Check either file is write-able or not
+    if not access(value, W_OK):
+        raise OptionValueError('no write access to "%s"' % value)
+    setattr(parser.values, option.dest, value)
+
 def obsolete_option(option, opt_str, value, parser):
     raise parser.error('"%s" now obsolete' % option)#}}}
 
@@ -129,7 +136,7 @@ OPTS = list([
         'type': 'string',
         'dest': 'var_dir',
         'action': 'callback',
-        'callback': is_file_rwable,
+        'callback': is_directory_rwable,
         'help': 'Directory to store var content (ie. database)'},
     {
         'sw': ['--new-version'],
